akd/src/storage/tests.rs

```diff
@@ -48,7 +48,6 @@ mod memory_storage_tests {
 /// This is public because it can be used by other implemented storage layers
 /// for consistency checks (e.g. mysql, memcached, etc)
 pub async fn run_test_cases_for_storage_impl<S: Storage + Sync + Send>(db: &S) {
-    crate::test_utils::init_logger(log::Level::Info);
```

Comment:
Do we need to enable the logger in the individual tests inside our storage tests if this isn't here?

Reply:
Oh good catch! I don't have to enable it for storage tests within the AKD crate - `ctor` does that for us, but I have to enable it for the test in the `akd_mysql` crate.